biomaj/workflow.py

```diff
@@ -1692,7 +1692,10 @@ def wf_uncompress(self):
 
                 not_ok = True
                 while nb_try < 3 and not_ok:
-                    status = Utils.uncompress(origFile)
+                    try:
+                        status = Utils.uncompress(origFile)
+                    except Exception as e:
+                        logging.exception('Workflow:wf_uncompress:Exception:' + str(e))
```

Comment:
Did you test an exception here? If exception is raised, i am afraid that status variable, tested right after will be unknown as initialized in the try scope.

I think that status should be initialized in this case to False before the try